app/screens/conversion-flow/use-convert-money-details.ts

```diff
@@ -0,0 +1,95 @@
+import { DisplayCurrency, MoneyAmount, WalletOrDisplayCurrency } from "@app/types/amounts"
+import React from "react"
+import { usePriceConversion } from "../../hooks/use-price-conversion"
+import { WalletCurrency } from "@app/graphql/generated"
+
+type Wallet = {
+  balance: number
+  walletCurrency: WalletCurrency
+  id: string
+}
+
+export const useConvertMoneyDetails = () => {
+  const [wallets, _setWallets] = React.useState<{
+    fromWallet: Wallet
+    toWallet: Wallet
+  }>()
+  const [moneyAmount, setMoneyAmount] = React.useState<
+    MoneyAmount<WalletOrDisplayCurrency>
+  >({ amount: 0, currency: DisplayCurrency })
+
+  const setWallets = (wallets: { fromWallet: Wallet; toWallet: Wallet }) => {
+    if (wallets.fromWallet.balance === 0) {
```

Comment:
maybe a comment explaining the intent of this line can be helpful? I guess we want to have the fromWallet having the funds but this requires me to read the line several time to understand the idea.

what happens if both wallets have 0 balance?

Reply:
I can add a comment. If both wallets have 0, it will still show them both but you wont be able to do anything with them.